pyranges/get_fasta.py

```diff
@@ -1,10 +1,9 @@
-
 import sys
 import pyranges as pr
 import pandas as pd
 
-def get_fasta(gr, path=None, pyfaidx_fasta=None):
```

Comment:
This breaks backward compatibility. I'm OK with the rename, but I'd prefer to create an alias like `get_fasta = get_sequence`.

Reply:
I had added a new def line for get_fasta with:
`def get_fasta(*args, **kwargs):
    """ Deprecated: this function has been moved to Pyranges.get_sequence"""
    return get_sequence(*args, **kwargs)`

Basically to ensure backward compatibility and also still have an entry in the documentation